app/controllers/MovementsController.scala

```diff
@@ -144,7 +146,10 @@ class MovementsController @Inject()(
         }
   }
 
-  def getArrival(arrivalId: ArrivalId): Action[AnyContent] = defaultActionBuilder(_ => NotImplemented)
+  def getArrival(arrivalId: ArrivalId): Action[AnyContent] = authenticatedArrivalForRead(arrivalId) {
+    implicit request =>
+      Ok(Json.toJsObject(request.arrival))
```

Comment:
This won't be the correct json response which would be using this writes: https://github.com/hmrc/transit-movements-trader-at-destination/blob/master/app/models/Arrival.scala#L58-L68

Reply:
That's fair - in which case this will also apply for the getArrivals endpoint. Do you want me to apply the ResponseArrival model to that as well?